lib/actions/ssh.coffee

```diff
@@ -53,6 +54,10 @@ module.exports =
 			boolean: true
 			description: "don't use the proxy configuration for this connection.
 				Only makes sense if you've configured proxy globally."
+	,
+			signature: 'nosuggest'
+			boolean: true
+			description: 'do not automatically suggest devices to connect to'
```

Comment:
The suggestion of devices to connect to -- is this the only interactive action taken by balena ssh? Does it perhaps "allocate a tty" (for the CLI process, if this makes sense) when this is unnecessary in the context of a script?
The reason for these questions is that, as you know, there is the idea/wish of having a "scriptable" or "non interactive" mode across all CLI commands... Here it's only for `balena ssh`, of course, but perhaps the name could be a bit more generic, like '--interactive off' or '--noninteractive' or '--scriptable' or something in these lines. But 'nosuggest' is OK too. :-)

Reply:
@pdcastro not sure what else is using interactive mode (maybe configure?) but this one is a big pain for me.

You are right, let's change the flag, will have something more generic in a bit.

Reply:
Sorry, I meant whether any other aspect or option of `balena ssh` is interactive, not whether other CLI commands are interactive. Focusing only on `balena ssh` here... I don't expect you to fix the whole CLI. :-)